src/Sylius/Bundle/UiBundle/Resources/translations/flashes.en.yml

```
@@ -22,9 +22,11 @@ sylius:
         apply_state_machine_transition_error: 'There was an unexpected problem with updating a payment. Please, try to update payment again.'
     product:
         out_of_stock: 'There is only %quantity% of this product in the stock.'
+        can_not_delete_product_assigned_to_promotion_rule: 'You cannot delete a product that is assigned to a promotion rule.'
```

```suggestion
        used_in_promotion_rule: 'Cannot delete a product that is used in a promotion rule.'
```
wdyt? similar w/ taxon